src/node.cc

```diff
@@ -4312,8 +4321,10 @@ int Start(int argc, char** argv) {
   V8::SetEntropySource(crypto::EntropySource);
 #endif
 
+#ifndef NODE_EMBEDDED_MODE
   default_platform = v8::platform::CreateDefaultPlatform(v8_thread_pool_size);
```

Comment:
If we don't use `v8_thread_pool_size`, then arguably the `--v8-pool-size=` option should cease to exist.  That makes for even more ugly #ifndefs, though...